apps/portals/src/portal-components/RouteControl.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useLayoutEffect, useRef } from 'react'
+import {
+  Tab,
+  Tabs,
+  TabScrollButton,
+  TabScrollButtonProps,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material'
+
+export type RouteControlProps = {
+  handleChanges: (text: string, index: number) => void
+  isSelected: (name: string) => boolean
+  customRoutes: string[]
+}
+
+function CustomScrollButton(props: TabScrollButtonProps) {
+  if (props.disabled) {
+    return <></>
+  }
+  return (
+    <TabScrollButton
+      {...props}
+      sx={{
+        svg: {
+          color: 'secondary.main',
+          fontSize: '26px',
+        },
+        '&:hover': {
+          backgroundColor: 'grey.300',
+        },
+      }}
+    />
+  )
+}
+
+export function RouteControl(props: RouteControlProps) {
```

Comment:
We had separate mobile/desktop components for responsiveness. Now just using MUI breakpoints to change styling/props.